fxcore/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "fxcore",
+  "chain_id": "fxcore",
+  "website": "https://functionx.io",
+  "pretty_name": "f(x)Core",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "fx",
+  "daemon_name": "fxcored",
+  "node_home": "$HOME/.fxcore",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "FX",
+        "fixed_min_gas_price": 4200000000000,
+        "low_gas_price": 400000000000,
+        "average_gas_price": 4200000000000,
+        "high_gas_price": 500000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "FX"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/functionx/fx-core",
+    "recommended_version": "v4.2.1",
```

Comment:
Can you please copy these codebase properties (all that you have here except for `git_repo` and `genesis`) into an object within a `versions` array? We are trying to ensure there exists a versions array for all chains and build up a version history for each over time.